web/operator/dist/build.js

```diff
@@ -0,0 +1,4 @@
+// Copyright (C) 2019 Storj Labs, Inc.
```

Comment:
Is there a reason this folder is called `operator` and not `storagenode`?

Reply:
@egonelbre this folder is created only for testing purposes. @NikolaiYurchenko  and @VitaliiShpital  are responsible for frontend part and they used exactly this name as you mentioned. But they are blocked by this pull request